### PR TITLE
Misc edits

### PR DESCRIPTION
**Pull request by antoineok**
Fixing two typos in the build command in the examples README
Modifying the Cargo.toml to use the local ogc_sys instead of re downloading the whole repo.

### CODE REVIEW

**Review by Daxorinator — COMMENTED** [review]
Thanks for the PR! This library doesn't get a lot of love so we appreciate you doing some housekeeping for us.

I'm just waiting on Frictionless (who knows more about ogc-sys than I do) to confirm that this won't break certain workflows before we approve this :D

File: Cargo.toml [hunk 1]
```diff
@@ -26,7 +26,7 @@ cfg-if = "1.0"
 cstr_core = "0.2"
 libm = "0.2"
 libc = "0.2"
-ogc-sys =  { git = "https://github.com/rust-wii/ogc-rs.git"}
+ogc-sys =  { path = "./ogc-sys/"}
```
[the remark this answers is not in the document]
Comment (Daxorinator) [reply Cargo.toml:29]: Even though the build didn't pass, it _did_ find ogc-sys without a problem so I'm happy to say this probably won't break anything

**Comment by antoineok** [comment]
oh yes, and the checks failed because libogc now is on version 2.4.1, which changes things about AESND methods

**Comment by Daxorinator** [comment]
Yeah exactly, I'll probably just bypass checks and commit this anyways because I'm happy enough your change wouldn't cause a build fail (if it was actually compiling lol)

**Comment by antoineok** [comment]
but since i don't really know anything about wii homebrew, i don't want to mess things up by making important changes to aesnd.rs

**Pull request merged**